.github/workflows/ci.yml

```diff
@@ -82,7 +82,6 @@ jobs:
           composer box-install
           # Warm the symfony cache so it gets bundled with phar.
           ./bin/acli
-          composer dump-env prod
```

Comment:
Turns out this does nothing because we don't use the framework bundle. Can't load .env files, etc.